File: vane-proxy-core/src/main/java/org/oddlama/vane/proxycore/ProxyPlayer.java
package org.oddlama.vane.proxycore;

import java.util.UUID;
import org.oddlama.vane.proxycore.commands.ProxyCommandSender;

public interface ProxyPlayer extends ProxyCommandSender {
    void disconnect(String message);

    UUID getUniqueId();

    long getPing();
}

File: vane-proxy-core/src/main/kotlin/org/oddlama/vane/proxycore/ProxyPlayer.kt
package org.oddlama.vane.proxycore

import org.oddlama.vane.proxycore.commands.ProxyCommandSender
import java.util.UUID

interface ProxyPlayer : ProxyCommandSender {
    fun disconnect(message: String?)

    val uniqueId: UUID?

    val ping: Long
}
